appserver/jdbc/jdbc-runtime/src/main/java/org/glassfish/jdbc/deployer/DataSourceDefinitionDeployer.java

```diff
@@ -1060,7 +1061,7 @@ public String getPropertyValue(String name, String defaultValue) {
             String value = null;
             value = (String) desc.getProperties().get(name);
             if (value != null) {
-                return value;
+                return TranslatedConfigView.expandValue(value);
```

Comment:
Shouldn't this also be applied to the `getProperty(String name)` and `getPropertyValue(String name)` methods?

Reply:
The handling of getting the DataSourceDefinition properties are handled here https://github.com/payara/Payara/blob/745aa8290080e627015011070cb98a16a94414e8/appserver/jdbc/jdbc-runtime/src/main/java/org/glassfish/jdbc/deployer/DataSourceDefinitionDeployer.java#L575. If you look at all the payara related properties, they all use the getPropertyValue(String name, String defaultValue) method